backend/app/routers/brand.py

```
@@ -111,3 +94,21 @@ async def update_brand(
     db.commit()
     db.refresh(db_brand)
     return db_brand
+
+
+@router.get("/brands/{brand_id}")
+async def get_brand(
```

Ruff (0.8.2)

Missing return type annotation for public function `get_brand`

(ANN201)